x-pack/filebeat/module/panw/_meta/kibana/7/dashboard/Filebeat-panw-network-overview.json

```diff
@@ -119,7 +119,7 @@
           }
         ],
         "timeRestore": false,
-        "title": "[Filebeat palo_alto] Network Flows ECS",
+        "title": "[Filebeat panw] Network Flows ECS",
```

Comment:
Maybe we could use more human-friendly titles, like `[Filebeat Palo Alto Networks]...`, or upper case PANW (`[Filebeat PANW]...`)

Reply:
Right, it looks horrible now